pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -9,7 +9,80 @@
   <body>
     <script>
       $(function () {
-          $('[data-toggle="popover"]').popover({sanitize: false})
+          $('[data-toggle="popover"]').popover({sanitize: false});
+          $('.time-limit-edit-button').popover({
+              sanitize: false,
+              placement: 'right',
+              title: function() {
+                  let row = $(this).data('row');
+                  return row.action == 'set_time_limit_all' ? 'Change Time Limits' :
+                      row.open ? 'Change Time Limit' : 'Re-Open Instance';
+              },
+              container: 'body',
+              html: true,
+              trigger: 'click',
+              content: function() {
+                  let that = this;
+                  let row = $(this).data('row');
+                  let form = $('<form name="set-time-limit-form" method="POST">');
+                  let action = row.action ? row.action : 'set_time_limit';
+                  form.append(`<p>Total time limit: ${row.total_time}<br/>
+                                  Remaining time: ${row.time_remaining}
+                               </p>`);
+                  form.append(`<input type="hidden" name="__action" value="${action}">`);
+                  form.append('<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">');
+                  if (row.assessment_instance_id)
+                      form.append(`<input type="hidden" name="assessment_instance_id" value="${row.assessment_instance_id}">`);
+
+                  let select = $('<select class="form-control select-time-limit" name=plus_minus>');
+                  if (row.time_remaining_sec !== null) {
+                      if (row.has_open_instance) {
+                          select.append('<option value=+1>Add to instances with time limit</option>');
```

Comment:
Nit: I understand that omitting the `"` in attribute values is technically valid HTML but I'd rather we use it for consistency.